dev/test/index.ts

```diff
@@ -639,6 +639,58 @@ describe('instantiation', () => {
     );
   });
 
+  describe('preferRest configuration', () => {
+    let originalValue: string | undefined;
+
+    beforeEach(() => {
+      originalValue = process.env.FIRESTORE_PREFER_REST;
+    });
+
+    afterEach(() => {
+      if (originalValue === undefined) {
+        delete process.env.FIRESTORE_PREFER_REST;
+      } else {
+        process.env.FIRESTORE_PREFER_REST = originalValue;
+      }
+    });
+
+    it('preferRest is disabled (falsy) by default', async () => {
+      delete process.env.FIRESTORE_PREFER_REST;
+      const firestore = new Firestore.Firestore({});
+      expect(!!firestore['_settings'].preferRest).to.be.false;
```

Comment:
Added a comment to clarify. Curious what you think about alternatively initializing this setting value to `false` (instead of leaving it as undefined) in the case where the setting is not set by the consumer?

Reply:
IMO leaving it as `undefined` is preferred because it more clearly conveys that the setting is optional. Thanks for adding the comment; that is helpful (to me!).